lib/expression.js

```diff
@@ -794,8 +813,27 @@ CronExpression.parse = function parse(expression, options, callback) {
       }
 
       mappedFields.dayOfMonth = mappedFields.dayOfMonth.filter(function(dayOfMonth) {
+        if(dayOfMonth === 'L') {
+          return true;
+        }
         return dayOfMonth <= daysInMonth;
       });
+      //sort
+      mappedFields.dayOfMonth.sort((a,b) => {
+        let aIsNumber = typeof a === 'number';
+        let bIsNumber = typeof b === 'number';
```

Comment:
```suggestion
        var aIsNumber = typeof a === 'number';
        var bIsNumber = typeof b === 'number';
```

Just for backwards compability.